src/hooks/useActionCenterSubscribers/useSwapActionSubscriber.tsx

```diff
@@ -121,9 +121,7 @@ export const useSwapActionSubscriber = () => {
 
       const { chainId, account: address } = fromAccountId(swap.sellAccountId)
 
-      const swapBuyAsset = swap.buyAsset
-      const feeAsset = selectFeeAssetByChainId(store.getState(), swapBuyAsset.chainId)
-      const txHash = swap.metadata.relayerTxHash ?? swap.sellTxHash ?? buyTxHash
+      const txHash = swap.metadata.relayerTxHash ?? buyTxHash ?? swap.sellTxHash
```

Comment:
q: non-blocking but wondering if things will work for the special case of Butter relayer Txs?
Unable to test because of the min amount for Butter but looks like they wouldn't, since intermediary Tx is not the sell chain, but it is not the buy chain either, so `defaultExplorerBaseUrl` logic below wouldn't work. Not a huge concern and tbh, probably better to do in a follow-up than now for the sake of progression.

Or perhaps we do one simpler and don't even care about relayerTxHash?

 ```suggestion
      const txHash = buyTxHash ?? swap.sellTxHash
```

Reply:
Confirmed that's borked as per *code intent*:

https://jam.dev/c/b946879b-5e7f-49af-9525-c29ba26fd2a2

 
<img width="1126" height="994" alt="Screenshot 2025-07-23 at 13 34 06" src="https://github.com/user-attachments/assets/08edcf65-d4f2-4b59-adc3-140d42e3a5ae" />
<img width="1128" height="994" alt="Screenshot 2025-07-23 at 13 34 00" src="https://github.com/user-attachments/assets/9446ce44-0015-463e-ab20-cffcda7951f8" />

With that being said, as mentioned above, that's not a bug for *actual* users, just the code not performing what it should. Honestly I'd just remove the `relayerTxHash` from notification center subscriber (i.e this hook), super useless IMO. Nice to have for swapper with multiple Txs, not so much here IMO.